utils/useProcessGame.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback } from 'react';
+import { getGameData, getUserPick } from './utils';
+import { useDataStore } from '@/store/dataStore';
+
+interface UseProcessGameProps {
+  leagueId: string;
+  gameWeek: { id: string };
+  user: { id: string };
+  NFLTeams: any; // Define this type based on your actual data structure
+  setUserPick: (data: any) => void; // Define this type based on your actual data structure
```

Comment:
No use of any. Please create or find the correct type.

Reply:
✅ fixed:

```js
interface UseProcessGameProps {
  leagueId: string;
  gameWeek: { id: string };
  user: { id: string };
  NFLTeams: INFLTeam[];
  setUserPick: (data: string) => void; // eslint-disable-line
}
```

note es-lint yells at unused variables in type declarations